docs/getting-started.md

```diff
@@ -773,6 +773,8 @@ against real data.  [Read more about permissions here](permissions.md).
 In the meantime, take a look at the rest of our documentation for more
 detail on the subjects covered in this tutorial. For example:
 
+* There is a [more complete guide](opensafely-cli.md) to the OpenSAFELY
```

Comment:
as part of the accessibility guidelines, I think we need to make sure we're using meaningful link text - I don't think `[more complete guide] to the cli` stands on its own well enough as link text, how about `more complete [guide to the cli]`?

Reply:
Yep, agree!